docs/gaia/delegator-guide-cli.md

```diff
@@ -0,0 +1,405 @@
+
+- `<yourKeyName>` is the name of the account. It is a reference to the account number used to derive the key pair from the mnemonic. You will use this name to identify your account when you want to send a transaction.
+- You can add the optional `--account` flag to specify the path (`0`, `1`, `2`, ...) you want to use to generate your account. By default, account `0` is generated. 
+
+### Creating an account
+
+To create an account, you just need to have `gaiacli` installed. Before creating it, you need to know where you intend to store and interract with your private keys. The best options are to store them in an offline dedicated computer or a ledger device. Storing them on your regular online computer involves more risk, since anyone who infiltrates your computer through the internet could exfiltrate your private keys and steal your funds.
+
+#### Using a ledger device
+
+When you initialize your ledger, a 24-word mnemonic is generated and stored in the device. This mnemonic is compatible with Cosmos and Cosmos accounts can be derived from it. Therefore, all you have to do is make your ledger compatible with `gaiacli`. To do so, you need to go through the following steps:
+
+1. Download the Ledger Live app [here](https://www.ledger.com/pages/ledger-live)
+2. Connect your ledger via USB and update to the latest firmware
+3. Go to the ledger live app store, and download the "Cosmos" application (this can take a while)
+4. Navigate to the Cosmos app on your ledger device
+
+Then, to create an account, use the following command:
+
+```bash
+gaiacli keys add <yourAccountName> --ledger 
+```
+
+- `<yourKeyName>` is the name of the account. It is a reference to the account number used to derive the key pair from the mnemonic. You will use this name to identify your account when you want to send a transaction.
+- You can add the optional `--account` flag to specify the path (`0`, `1`, `2`, ...) you want to use to generate your account. By default, account `0` is generated. 
+
+#### Using a computer 
+
+::: warning
+**NOTE: It is more secure to perform this action on an offline computer**
+:::
+
+To generate an account, just use the following command:
+
+```bash
+gaiacli keys add <yourKeyName>
+```
+
+The command will generate a 24-words mnemonic and save the private and public keys for account `0` at the same time. You will be prompted to input a passphrase that is used to encrypt the private key of account `0` on disk. Each time you want to send a transaction, this password will be required. If you lose the password, you can always recover the private key with the mnemonic. 
+
+::: danger
+**NEVER LOSE YOUR MNEMONIC. WRITE IT DOWN ON A PIECE OF PAPER AND STORE IT SOMEWHERE SAFE. IF YOU LOSE IT, THERE IS NO WAY TO RETRIEVE IT. IF SOMEONE GAINS ACCESS TO IT, THEY GAIN ACCESS TO ALL THE ASSOCIATED ACCOUNTS**
```

Comment:
(Same feedback as above)

It might be better to reword this as "Do not lose or share your 12 words with anyone. To prevent theft or loss of funds, it is best to ensure that you keep multiple copies of your mnemonic, and store it in a safe, secure place and that only you know how to access. If someone is able to gain access to your mnemonic, they will be able to gain access to your private keys and control the accounts associated with them."